src/common/actions/auth-store.js

```diff
@@ -270,7 +272,13 @@ async function setShortNamespace(root, discharge, userName) {
     }
     return data;
   } else {
-    const json = await response.json();
+    const text = await response.text();
```

Comment:
One reason why any service might not return valid JSON would be if it's down such that haproxy returns some kind of dumb text-only response.  We should always handle that (not saying that we do at the moment).